File: latihan-1/b/java/TimSepakbola.java
public class TimSepakbola {

    private String NamaTeam;
    private String NegaraAsalTeam;
    private int TahunBerdiriTeam;
    private String Pemain;
    private String NegaraAsalPemain;
    private int NomorPunggungPemain;
    private String PosisiPemain;

    public TimSepakbola(String NamaTeam, String NegaraAsalTeam, int TahunBerdiriTeam, String Pemain, String NegaraAsalPemain, int NomorPunggungPemain, String PosisiPemain){
        this.NamaTeam = NamaTeam;
        this.NegaraAsalTeam = NegaraAsalTeam;
        this.TahunBerdiriTeam = TahunBerdiriTeam;
        this.Pemain = Pemain;
        this.NegaraAsalPemain = NegaraAsalPemain;
        this.NomorPunggungPemain = NomorPunggungPemain;
        this.PosisiPemain = PosisiPemain;
    }

    public TimSepakbola(){}

    public void setNamaTeam(String NamaTeam){
        this.NamaTeam = NamaTeam;
    }

    public String getNamaTeam(){
        return NamaTeam;
    }

    public void setNegaraAsalTeam(String NegaraAsalTeam){
        this.NegaraAsalTeam = NegaraAsalTeam;
    }

    public String getNegaraAsalTeam(){
        return NegaraAsalTeam;
    }

    public void setTahunBerdiriTeam(int TahunBerdiriTeam){
        this.TahunBerdiriTeam = TahunBerdiriTeam;
    }

    public int getTahunBerdiriTeam(){
        return TahunBerdiriTeam;
    }

    public void setPemain(String Pemain){
        this.Pemain = Pemain;
    }

    public String getPemain(){
        return Pemain;
    }

    public void setNegaraAsalPemain(String NegaraAsalPemain){
        this.NegaraAsalPemain = NegaraAsalPemain;
    }

    public String getNegaraAsalPemain(){
        return NegaraAsalPemain;
    }

    public void setNomorPunggungPemain(int NomorPunggungPemain){
        this.NomorPunggungPemain = NomorPunggungPemain;
    }

    public int getNomorPunggungPemain(){
        return NomorPunggungPemain;
    }

    public void setPosisiPemain(String PosisiPemain){
        this.PosisiPemain = PosisiPemain;
    }

    public String getPosisiPemain(){
        return PosisiPemain;
    }
}

File: latihan-1/b/cpp/TimSepakbola.cpp
#include <iostream>
#include <string>
using namespace std;
class TimSepakbola{
    private:

        string NamaTeam, NegaraAsalTeam, Pemain, NegaraAsalPemain, PosisiPemain;
        int TahunBerdiriTeam, NomorPunggungPemain;
        
    public:

        TimSepakbola(){
            string NamaTeam = "";
            string NegaraAsalTeam = "";
            int TahunBerdiriTeam = 0;
            string pemain = "";
            string NegaraAsalPemain = "";
            int NomorPunggungPemain = 0;
            string PosisiPemain = "";
        }
        
        TimSepakbola(string NamaTeam, string NegaraAsalTeam, int TahunBerdiriTeam, string Pemain, string NegaraAsalPemain, int NomorPunggunPemain, string PosisiPemain){
            this->NamaTeam = NamaTeam;
            this->NegaraAsalTeam = NegaraAsalTeam;
            this->TahunBerdiriTeam = TahunBerdiriTeam;
            this->Pemain = Pemain;
            this->NegaraAsalPemain = NegaraAsalPemain;
            this->NomorPunggungPemain = NomorPunggungPemain;
            this->PosisiPemain = PosisiPemain;
        }

        void setNamaTeam(string NamaTeam){
            this->NamaTeam = NamaTeam;
        }

        string getNamaTeam(){
            return this->NamaTeam;
        }

        void setNegaraAsalTeam(string NegaraAsalTeam){
            this->NegaraAsalTeam = NegaraAsalTeam;
        }

        string getNegaraAsalTeam(){
            return this->NegaraAsalTeam;
        }

        void setTahunBerdiriTeam(int TahunBerdiriTeam){
            this->TahunBerdiriTeam = TahunBerdiriTeam;
        }

        int getTahunBerdiriTeam(){
            return this->TahunBerdiriTeam;
        }

        void setPemain(string Pemain){
            this->Pemain = Pemain;
        }

        string getPemain(){
            return this->Pemain;
        }

        void setNegaraAsalPemain(string NegaraAsalPemain){
            this->NegaraAsalPemain = NegaraAsalPemain;
        }

        string getNegaraAsalPemain(){
            return this->NegaraAsalPemain;
        }

        void setNomorPunggungPemain(int NomorPunggungPemain){
            this->NomorPunggungPemain = NomorPunggungPemain;
        }

        int getNomorPunggungPemain(){
            return this->NomorPunggungPemain;
        }

        void setPosisiPemain(string PosisiPemain){
            this->PosisiPemain = PosisiPemain;
        }

        string getPosisiPemain(){
            return this->PosisiPemain;
        }

        ~TimSepakbola(){}
};